source/sdk/TextBuffer.ooc

```diff
@@ -20,11 +20,7 @@ import OwnedBuffer
 
 TextBuffer: cover {
 	_backend: OwnedBuffer
-	raw: Char* { get {
-		result := this _backend pointer as Char*
-		this free(Owner Receiver)
-		result
-	}}
+	raw ::= this _backend pointer as Char*
```

Comment:
This is not valid for property which should return a pointer to internal buffer. Please provide a single use case where this is correct to free a buffer and then return pointer to it.